### DISCUSSION TITLE
ChatGPT를 활용한 글 요약 기능

### DISCUSSION THREAD

**Discussion by slowhigh** [thewoowon/dentium.tech#59] [Ideas]
저희도 아래 이미지와 같이 ChatGPT를 활용해서 글에 대한 요약을 상단에 넣어 보는 건 어떤가요?

![image](https://github.com/thewoowon/dentium.tech/assets/37216082/8faa2066-4034-474e-b989-94aa59df8b5e)

출처: https://devocean.sk.com/experts/techBoardDetail.do?ID=165186&boardType=experts&page=&searchData=&subIndex=&idList=

**Comment by thewoowon** [comment]
너무 좋습니다! ㅎㅎ
그럼 제가 오늘은 해커톤 때문에 힘들 것 같고,
내일 적용해서 공유하도록 하겠습니다.

그리고 신기한게 제가 지금 SK 텔레콤 데보션 운영자분이랑 같이 있는데...ㅋㅋㅋㅋ
신기합니다

제가 이분한테 물어 보고 저희 블로그랑 콜라보도 물어 보겠습니다.
감사합니다.

-----Original Message-----
From: "Jo ***@***.***>
To: ***@***.***>;
Cc: ***@***.***>;
Sent: 2023-09-23 (토) 20:36:28 (GMT+09:00)
Subject: [thewoowon/dentium.tech] ChatGPT를 활용한 글 요약 기능 (Discussion #59)

저희도 아래 이미지와 같이 ChatGPT를 활용해서 글에 대한 요약을 상단에 넣어 보는 건 어떤가요?
출처: https://devocean.sk.com/experts/techBoardDetail.do?ID=165186&boardType=experts&page=&searchData=&subIndex=&idList=
